apps/dashboard/src/pages/inbox-embed-page.tsx

```diff
@@ -0,0 +1,38 @@
+import { AuthCard } from '../components/auth/auth-card';
+import { ROUTES } from '../utils/routes';
+import { InboxEmbed } from '../components/welcome/inbox-embed';
+import { UsecasePlaygroundHeader } from '../components/usecase-playground-header';
+import { useTelemetry } from '../hooks/use-telemetry';
+import { TelemetryEvent } from '../utils/telemetry';
+import { useEffect } from 'react';
+
+export function InboxEmbedPage() {
+  const telemetry = useTelemetry();
+
+  useEffect(() => {
+    telemetry(TelemetryEvent.INBOX_EMBED_PAGE_VIEWED);
+  }, [telemetry]);
+
+  return (
+    <AuthCard className="mt-10 w-full max-w-[1230px]">
+      <div className="w-full">
+        <div className="flex flex-1 flex-col overflow-hidden">
+          <UsecasePlaygroundHeader
```

Comment:
So, it's a little bit intended because you have already connected your inbox previously. This is persisted on our integration store as connected. Once connected you are always redirected here. Does it make sense?

Reply:
The previous step is about sending the Inbox notification. IDK, I feel like we should allow navigating between the steps. The last one with confetti is fine to show when there is connected integration.

Reply:
Oh, it does work, the code navigated using `navigate(-1) to the previous page you came from. Could it be that you navigated to it from the welcome page? I've just tested the onboarding flow and it brought me back here:
![Uploading CleanShot 2024-12-04 at 10.42.28@2x.png…]()